lib/index.ts

```diff
@@ -15,8 +15,9 @@ import preferPresenceQueries from './rules/prefer-presence-queries';
 import preferScreenQueries from './rules/prefer-screen-queries';
 import preferUserEvent from './rules/prefer-user-event';
 import preferWaitFor from './rules/prefer-wait-for';
-import noMultipleAssertionsWaitFor from './rules/no-multiple-assertions-wait-for'
+import noMultipleAssertionsWaitFor from './rules/no-multiple-assertions-wait-for';
 import preferFindBy from './rules/prefer-find-by';
+import invalidRenderResultName from './rules/render-result-naming-convention';
```

Comment:
Looking at the other rules, all them are following the pattern to name the import as the rule name. Any reason to change here? Maybe could be good to keep the same pattern

Reply:
Oops, I think I changed this one in a rename that shouldn't affect it. I'll change it.